projects/1.16.1/assets/astral-sorcery/astralsorcery/lang/zh_cn.json

```
@@ -137,20 +137,20 @@
   "screen.astralsorcery.tome.progression": "研究",
   "screen.astralsorcery.tome.constellations": "星座",
   "screen.astralsorcery.tome.perks": "星能力",
-  "screen.astralsorcery.tome.perks.stats": "技能统计数据",
+  "screen.astralsorcery.tome.perks.stats": "星能力统计数据",
   "screen.astralsorcery.tome": "宝典",
 
-  "key.astralsorcery.disable_perk_abilities": "切换技能和工具能力",
-
-  "astralsorcery.progress.missing.knowledge": "这似乎记载了什么，以当前的知识无法理解。",
-  "astralsorcery.progress.gain.progress.chat": "你的学识增长了。",
-  "astralsorcery.progress.gain.research.chat": "你对'%s'更了解了。",
-  "astralsorcery.progress.constellation.discover.chat": "发现%s星座！",
-  "astralsorcery.progress.constellation.seen.chat": "你已经记住了%s的内容！",
-  "astralsorcery.progress.constellation.seen.track": "你可以在星芒宝典中加入一个标签来记录它。",
-  "astralsorcery.progress.remove.attunement": "你的星能力消逝了……",
-  "astralsorcery.progress.switch.attunement": "星能力被巧妙地转换了……",
-  "astralsorcery.progress.perk_abilities": "技能和工具能力力现已为 %s",
+  "key.astralsorcery.disable_perk_abilities": "切换星能力和工具能力",
+
+  "astralsorcery.progress.missing.knowledge": "这上面似乎记载了什么，但以你当前的知识无法理解。",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "astralsorcery.progress.missing.knowledge": "这上面似乎记载了什么，但你以当前的知识尚无法理解。",

```